Makefile

```diff
@@ -8,8 +8,8 @@ DOCKER_REGISTRY_SIDECARS := quay.io/k8scsi
 # if you pull from public use quay.io/k8scsi
 PULL_SIDECARS := true
 
-DATASET_OPERATOR_NAMESPACE := default
-NAMESPACES_TO_MONITOR := default
+DATASET_OPERATOR_NAMESPACE := drlmcdlf
+NAMESPACES_TO_MONITOR := drlmcdlf
```

Comment:
And these too :smiley:

Reply:
fixed 9c66ac156452e7d4c3fd366a85116342b694b03f